code/src/components/Loader.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import './Loader.css'
+
+const Loader = () => {
+
+    return (
+		<div className="loader-overlay">
+            <h1>HAPPY LOADING . . .</h1>
+			<div className="loading-heart" />
```

Comment:
The indentions are not quite aligned here

Reply:
Right! didn't put the time to go through it. Will  think about that! Thank you for reminding me 😃